fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -348,7 +360,7 @@ class ReadIteratorPool {
 	}
 
 	// Called on every read operation.
-	ReadIterator getIterator() {
+	ReadIterator getIterator(const KeyRange& range) {
```

Comment:
We are not checking if an existing iterator has the correct range if `ROCKSDB_READ_RANGE_REUSE_ITERATORS`?

Reply:
OK, nvm, this bound is turned on only if `ROCKSDB_READ_RANGE_REUSE_ITERATORS` is off, right?